stackrox/stackrox
# chore(ui): Delete ROX_ADMINISTRATION_EVENTS feature flag

## Description

Delete counterpart to add in #7252

https://github.com/stackrox/stackrox/tree/master/ui/apps/platform#delete-a-feature-flag-from-frontend-code

Do **not** delete from list.go nor lib.sh files, because backend code still uses feature flag.

Feature flag renamed and turned on since 2023-09-14 in #7594

## Checklist

## Testing Performed

1. `yarn lint` in ui
2. `yarn build` in ui and then compute branch - master for the following
    * `wc build/static/js/*.js`
        main.js -63 = 4636841 - 4636904
        total -228 = 11722156 - 11722384
    * `ls -al build/static/js/*.js | wc`
        files 0 = 174 - 174
3. `yarn start` in ui

### Manual testing

1. Visit /main/administration-events
    ![routePaths](https://github.com/stackrox/stackrox/assets/11862657/09d4b5e8-f0de-4bae-9e76-b8179d40de88)

2. Visit /main/systemconfig and see **Administration events retention days**
    ![systemconfig_view](https://github.com/stackrox/stackrox/assets/11862657/c4ee5260-c51c-4a16-bc1b-0e0cec182274)

3. Click **Edit** and see **Administration events retention days**
    ![systemconfig_edit](https://github.com/stackrox/stackrox/assets/11862657/dc7c9db3-4401-480e-85d3-219f0abe1623)

### Integration testing

Without `CYPRESS_ROX_ADMINISTRATION_EVENTS` feature flag.

1. `yarn cypress-open` in ui/apps/platform
    * administration/events/eventsTable.test.js
